src/tests/excalidrawPackage.test.tsx

```diff
@@ -135,61 +135,55 @@ describe("<Excalidraw/>", () => {
     it('should not hide any UI element when the UIOptions prop is "undefined"', async () => {
       const { container } = await render(<Excalidraw />);
 
-      expect(queryByTestId(container, "clear-canvas-button")).toBeTruthy();
-      expect(queryByTestId(container, "export-button")).toBeTruthy();
-      expect(queryByTestId(container, "load-button")).toBeTruthy();
-      expect(queryByTestId(container, "save-as-button")).toBeTruthy();
-      expect(queryByTestId(container, "save-button")).toBeTruthy();
+      expect(container).toMatchSnapshot();
```

Comment:
not the entire container, i meant just the snapshot of canvas actions div as thats only needed in both cases.

Reply:
canvas actions div won't have all actions after #3381. What should we do?

Reply:
We can take the snap of the bottom container as well when the pr is merged. for this PR, the canvas div snapshot should be fine.